src/java.base/share/classes/java/util/Base64.java

```diff
@@ -810,12 +810,12 @@ private int decode0(byte[] src, int sp, int sl, byte[] dst) {
 
             while (sp < sl) {
                 if (shiftto == 18 && sp < sl - 4) {       // fast path
-                    int dl = decodeBlock(src, sp, sl, dst, dp, isURL);
+                    int dl = decodeBlock(src, sp, sl, dst, dp, isURL, isMIME);
```

Comment:
This new param is passed all the way down to the intrinsic.  I think existing intrinsics can safely ignore this parameter if it doesn't help the implementation (for example PPC64-LE has 16-byte vector registers, so isn't quite as seriously impacted by MIME).  However, in the code for the PPC64-LE intrinsic, this new parameter isn't mentioned.  I think if you're going to add a new parameter, it should be mentioned in the existing intrinsics as being present, but unused.

Reply:
Are you suggesting that I change *all* intrinsic implementations (aarch64, ppc, etc.)?  I have no problem doing that - just checking if this is what's desired.

Reply:
Yes. I didn't realize that there's a decodeBlock intrinsic for aarch64 already, but yeah it should only be a couple of lines of comments for each.